wiki/Tournaments/CFB/4/fr.md

```diff
@@ -133,8 +135,8 @@ La Catch French Beginner 4 ést gérée par différents membres de la communaut
 | #15 | ![][flag_FR] [Kammthaar](https://osu.ppy.sh/users/8802523) | 14.4 |
 | #16 | ![][flag_FR] [sullymaster](https://osu.ppy.sh/users/6765259) | 16 |
 
-- Les résultats plus détaillés des qualificatifs sont accessibles via la fiche statistique dans la [section dla posites liens](#liens).
-- Le classement moyen est déterminé par `la position obtenue sur Σ en jouant le mappage des qualifications / le nombre de mappages des qualifications`.
+- Plus de détails concernant les résultats des qualifications sont accessibles dans la [section des liens](#liens) via la feuille des statistiques.
+- Le rang moyen est déterminé par : `Somme des positions du joueur sur chaque map divisée par le nombre de maps composant le mappool des qualifications`.
```

Comment:
it doesn't need the code blocks if it's not a mathematical expression anymore. but I think you should try to match english with the `Σ` and `/` ...